src/accounts/CancellationOverlay.tsx

```diff
@@ -0,0 +1,146 @@
+// Libraries
+import React, {FC, useContext, useMemo, useState} from 'react'
+import {
+  Overlay,
+  Alert,
+  ComponentColor,
+  ComponentSize,
+  IconFont,
+  Button,
+  ComponentStatus,
+} from '@influxdata/clockface'
+
+// Components
+import CancellationTerms from './CancellationTerms'
+import {
+  CancelServiceContext,
+  VariableItems,
+} from 'src/billing/components/PayAsYouGo/CancelServiceContext'
+import {track} from 'rudder-sdk-js'
+import {event} from 'src/cloud/utils/reporting'
```

Comment:
Most of this code is copied from the existing PayAsYouGo code base. Just the terms wording has been changed a little to match the new requirements.